docs/excel/custom-functions-authentication.md

```diff
@@ -39,6 +39,37 @@ The following diagram outlines this basic process. Note that the dotted line ind
 
 ![Diagram of custom functions, OfficeRuntime, and task panes working together.](../images/Authdiagram.png "Authentication diagram.")
 
+## Storing the token
+
+The following examples are from the [Using AsyncStorage in custom functions](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Excel-custom-functions/AsyncStorage) code sample. Refer to this code sample for a complete example of sharing data between custom functions and the task pane.
+
+If the custom function authenticates, then it receives the access token and will need to store it in AsyncStorage. The following code sample shows how to call the `AsyncStorage.setItem` method to store a value. The `StoreValue` function is a custom function that for example purposes stores a value from the user. You can modify this to store any token value you need.
+
+```javascript
+function StoreValue(key, value) {
+  return OfficeRuntime.AsyncStorage.setItem(key, value).then(function (result) {
+      return "Success: Item with key '" + key + "' saved to AsyncStorage.";
+  }, function (error) {
+      return "Error: Unable to save item with key '" + key + "' to AsyncStorage. " + error;
+  });
+}
+```
+
+When the task pane needs access, it can retrieve the access token from AsyncStorage. The following code sample shows how to use the `AsyncStorage.getItem` method to retrieve the token.
```

Comment:
Suggest formatting `AsyncStorage` as code throughout this article (some instances are correctly formatted as code, but several are just plain text...in this section and elsewhere).

Reply:
fixed.